docs/visualisation/alchemlyb.visualisation.plot_convergence.rst

```diff
@@ -0,0 +1,19 @@
+.. _visualisation_plot_convergence:
+
+Plot the Forward and Backward Convergence
+=========================================
+
+The function :func:`~alchemlyb.visualisation.plot_convergence` allows
+the user to visualise the convergence by plotting the free energy change
+computed using the equilibrated snapshots between the proper target time frames
+in both forward (data points are stored in `forward` and `forward_error`) and
+reverse (data points are stored in `backward` and `backward_error`) directions.
+The unit in the y axis could be labelled to other units by setting *units*,
+which by default is :math:`kBT`. The user can pass :class:`matplotlib.axes.Axes` into
```

Comment:
```suggestion
which by default is :math:`k_B T` ("kT"). The user can pass :class:`matplotlib.axes.Axes` into
```

Can we make the string to indicate k_B T just "kT" instead of "kBT"? The latter puts too much emphasis on the "B" and for physicists at least, we're just talking about kT.

Reply:
I have change this to :math:`kT`

Reply:
@orbeckst Thank you for the explanation. In that case, I will change the doc for the function to 'kt' and the sphinx doc to :math:`kT` so that it is more clear.